src/components/NewLocationPage/ChartTabs/ChartTab.style.tsx

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components';
+import { COLOR_MAP } from 'common/colors';
+import { materialSMBreakpoint } from 'assets/theme/sizes';
+
+export const TabsContainer = styled.div`
+  display: flex;
+`;
+
+export const Tab = styled.div`
```

Comment:
for accessibility purposes, tabs should be buttons in the dom (they're clickable, they do an action, etc). check out [material ui tabs](https://material-ui.com/components/tabs/)--they have a lot of functionality and accessibility considerations built in

Reply:
and here is an example of mui tabs currently used in the code: [component](https://github.com/covid-projections/covid-projections/blob/develop/src/components/Explore/ExploreTabs.tsx) and [styles](https://github.com/covid-projections/covid-projections/blob/develop/src/components/Explore/Explore.style.ts#L44)